CredentialsManager/Models/Credentials.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+namespace CredentialsManager.Models
+{
+  public class Credentials : IEquatable<Credentials>, IComparable<Credentials>, IComparable
```

Comment:
Can be a struct instead of a class, because it is a `ValueObject` essentially, just 2 fields which you treat as 1.

Reply:
I successfully converted to it to struct in commit [4b30e14](https://github.com/csinn/CSharp-From-Zero-To-Hero-v2/pull/43/commits/4b30e147718194bba663095be2c2d4fa23a90759)

Reply:
Or even a `record` these days since it is immutable.